platform/android/java/editor/src/main/java/org/godotengine/editor/EditorMessageDispatcher.kt

```diff
@@ -173,7 +173,12 @@ internal class EditorMessageDispatcher(private val editor: GodotEditor) {
 		// to the sender.
 		val senderId = messengerBundle.getInt(KEY_EDITOR_ID)
 		val senderMessenger: Messenger? = messengerBundle.getParcelable(KEY_EDITOR_MESSENGER)
-		registerMessenger(senderId, senderMessenger)
+		registerMessenger(senderId, senderMessenger) {
+			// Terminate current instance when parent is no longer available.
+			// TODO: evaluate whether this is desired behavior.
+			Log.d(TAG, "Terminating current editor instance because parent is no longer available")
+			editor.finish()
+		}
```

Comment:
Just flagging the "TODO", as TODOs tend to not get done if not properly tracked :P

Reply:
I'll remove it. From testing I found it beneficial to automatically close the play window when the editor closes.